src/module.c

```diff
@@ -0,0 +1,101 @@
+#include "verify.h"
+#include <node_api.h>
+#include <stdio.h>
+
+napi_value Call_VF_verify(napi_env env, napi_callback_info info) {
+  napi_status status;
+  size_t argc = 6;
+  napi_value argv[argc];
+  status = napi_get_cb_info(env, info, &argc, argv, NULL, NULL);
+
+  if (status != napi_ok) {
+    napi_throw_error(env, NULL, "Failed to parse arguments");
```

Comment:
I couldn't find any examples where there was a return after calling a napi_throw* function, but you're right that it does continue C execution.  I've either removed all of my napi_throw* calls, added a return NULL/status where appropriate or left them without a return statement because there's nothing else happening in the function of import.